components/actionbar/index.css

```diff
@@ -184,3 +166,14 @@
 .spectrum-ActionBar--flexible .spectrum-ActionBar-popover {
 	inline-size: auto;
 }
+
+/* windows high contrast mode */
+@media (forced-colors: active) {
+	.spectrum-ActionBar {
+		--highcontrast-actionbar-popover-border-color: CanvasText;
+	}
+
+	.spectrum-ActionBar--emphasized .spectrum-ActionBar-popover {
+		--highcontrast-actionbar-popover-border-color: CanvasText;
```

Comment:
This selector is a little repetitive to me, so can we remove it? `--highcontrast-actionbar-popover-border-color: CanvasText;` was already declared above in `.spectrum-ActionBar`, and this second selector block is just variants, so isn't it already set?